python/cuml/benchmark/algorithms.py

```diff
@@ -124,6 +139,15 @@ def _labels_to_int_hook(data):
     return (data[0], data[1].astype(np.int32))
 
 
+def fit_kneighbors(m, x):
```

Comment:
Maybe these should be internal `_fit_kneighbors` functions so it's clear they're not a part of the external apis and are just internal utility wrappers?

Reply:
Actually, I was hoping to make these functions part of the public API- especially the fit_transform, since this is a common function in sklearn's Transformer API. Given the idea above, which would always use a bench_func, what do you think about making a separate file for the different funcs we use? Maybe something like bench_funcs.py?

Reply:
`bench_helper_funcs.py` maybe? Sounds good.

Reply:
Sounds good. I’ll rename the file.